test/image/mocks/range_slider_axes_stacked.json

```diff
@@ -0,0 +1,11 @@
+{
+  "data": [
+    {"y": [1, 2, 3], "yaxis": "y2"},
+    {"fill": "tozeroy", "y": [30, 20, 10]}
+  ],
+  "layout": {
+    "xaxis": {"rangeslider": {}, "title": "x"},
+    "yaxis": {"title": "y", "domain": [0, 0.25]},
+    "yaxis2": {"title": "y2", "domain": [ 0.3, 1]}
```

Comment:
Referencing https://github.com/plotly/plotly.js/issues/2172 and more specifically https://github.com/plotly/plotly.js/issues/2172#issuecomment-347279990 - so if I understand correctly, the baseline for this mock is _wrong_ as the desired behavior would be replicate the stacked y-axes in the range slider?

Reply:
yes, most likely this mock will change with #2172. Though reconciling #2172 with #2364 may be a bit of a challenge...